axon-framework/serialization.md

```diff
@@ -27,7 +37,22 @@ To that end, the `Configuration` API allows you to define default, message and e
    These objects are generally not shared between different applications, and most of these classes aren't expected to have some of the getters and setters that are, for example, typically required by Jackson based serializers.
    A flexible, general purpose serializer like [XStream](http://x-stream.github.io/) is quite suited for this purpose.
```

Comment:
```suggestion
   A flexible, general-purpose serializer like [XStream](http://x-stream.github.io/) is quite ideal for this purpose.
```
Sorry, I know I'm fixing text that was not modified. This is just a suggestion. :)

Reply:
No worries, Sara! The adjustment sounds very fair to me. Even if this isn't modified in this PR, we can still make it so.